index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+        <!--TOOLBAR SECTION-->
+        <nav class="toolbar">
+            <div class="my-logo primary-color hover-underline">My Logo</div>
+            <div class=" collapsed_menu primary-color">&#9776;</div>
+        </nav>
+        <!--END OF TOOLBAR SECTION-->
+
+        <!--HEADLINE SECTION-->
+        <section>
+            <div>
+                <div><img class="ellipse11" src="images/Ellipse11.jpg"></div>
+                <div class="ellipse_5"><img  src="images/Ellipse 5.jpg"></div>
+                <div class="rectangle_7"><img  src="images/Rectangle 7.jpg"></div>
+                <div class="rectangle_4"><img src="images/Rectangle 4.jpg"> </div>
+                <div class="ellipse_3"><img  src="images/Ellipse 3.jpg"></div>
+            </div>
+            <div class="greetings">
+                <h1>I am Naan.
+                    Glad to see you!
+                </h1>
+            </div>
+            <div class="description">
+                <p>I’m a software developer! I can help you build a product, 
+                    feature or website.Look through some of my work 
+                    and experience! If you like what you see and have a project  
+                    you need coded,don’t hesitate to contact me.
+                </p>
+            </div>
+            <div class="container-1">
+                <div class="lets_connect">
+                    <a href="www.github.com/naanahmed">LET'S CONNECT</a>
+                </div>
+                <div><img src="images/Ellipse 12.jpg"></div>
+            </div>
+            <div class="ellipse_13"><img src="images/Ellipse 13.jpg"></div>
+            <div class="img-container">
+                <div><img src ="images/twitter.jpg"></div>
+                <div><img src ="images/i.jpg"><img src ="images/n.jpg"></div>
+                <div><img src ="images/medium.jpg"></div>
+                <div><img src ="images/github.jpg"></div>
+                <div><img src ="images/reddit.jpg"></div>
+            </div>
```

Comment:
Kindly update this element to use a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each img in a `<li>` tag and make sure that the images are all links using an `<a>` tag. Your final code should look similar to something like this
`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`